docs/components/image.html

```diff
@@ -0,0 +1,140 @@
+                    <li class="sidebar-items-link"><a href="#">Input</a></li>
+                </ul>
+                <div class="sidebar-heading">Customization</div>
+                <ul class="sidebar-items">
+                    <li class="sidebar-items-link"><a href="#">Theme</a></li>
+                    <li class="sidebar-items-link"><a href="#">Palette</a></li>
+                    <li class="sidebar-items-link"><a href="#">Typography</a></li>
+                </ul>
+                <div class="sidebar-heading">Overview</div>
+                <ul class="sidebar-items">
+                    <li class="sidebar-items-link"><a href="#">Conclusion</a></li>
+                    <li class="sidebar-items-link"><a href="#">Get involved</a></li>
+                </ul>
+            </aside>
+
+            <main class="main-content">
+                <h1 class="main-head">Image</h1>
+                <p class="main-desc">Images that work well on devices with widely differing screen sizes and
+                    resolutions.
+                </p>
+                <div class="section-change"></div>
+
+                <!-- Responsive Image -->
+
+                <h3 class="main-subhead">Responsive Image</h3>
+                <p class="main-desc">To obtain this result, use <i>img</i> tag inside a <i>div</i> tag with
+                    <i>img-responsive</i> class name.
+                </p>
+                <div class="component-showcase rounded-5">
+                    <div>
+                        <img src="../../assets/image/image-5.jpg" alt="image" class="img-responsive">
+                    </div>
+                </div>
+                <div class="component-code-showcase">
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%2846%2C54%2C62%2C1%29&t=blackboard&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=10px&dsblur=26px&wc=true&wa=true&pv=41px&ph=47px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%253E%250A%2520%2520%2520%2520%253Cimg%2520src%253D%2522..%252F..%252Fassets%252Fimage%252Fimage-5.jpg%2522%2520alt%253D%2522image%2522%2520class%253D%2522img-responsive%2522%253E%250A%253C%252Fdiv%253E"
+                        style="width: 100%; height: 225px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+                <div class="section-change"></div>
+
+                <!-- Rounded image -->
+
+                <h3 class="main-subhead">Rounded Image</h3>
+                <p class="main-desc">To obtain this result, use <i>img</i> with
+                    <i>img-rounded</i> and <i>img</i> class name.
+                </p>
+                <div class="component-showcase rounded-5">
+                    <img src="../../assets/image/image-3.jpg" alt="image" class="img img-rounded">

```

Comment:
dont give classnames same as tag names. Not a good practice

Reply:
Okay will change it.